### ISSUE TITLE
Sites having 'Connected Apps' & 'Active Sessions' settings in the same page

### ISSUE THREAD

**Issue by Aravinth-Earth** [bilelmoussaoui/revoke.app#15]
For sites having the 'Connected Apps' & 'Active Sessions' settings in the same page, should the buttons be renamed/combined to 'Apps + Sessions' or similar?

For example below sites have similar design
* Quora
* Twitter

**Comment by bilelmoussaoui** [comment]
Indeed, we can easily do that by checking if both urls are the same and show on button instead of two.

**Comment by mmahmoudian** [comment]
It should be considered that the final button should be a merge of both concepts. I mean in some websites, both settings are on the same URL, but if you remove one button just because of URL redundancy, for example the Active Sessions, then revoke.app user might think that the website does not have a section to remove active sessions. I personally prefer to have an information twice rather than loosing a part of information.

**Comment by Aravinth-Earth** [comment]
@mmahmoudian 
If both settings are in same page, will show one button with the text labels combined like below,
![Quora - with buttons combined](https://user-images.githubusercontent.com/55803552/89735002-f0c57c80-da4e-11ea-9eb2-99599af30404.png)

And in cases if needed will append 'Devices' text as well. looks fine ?

@bilelmoussaoui 
That creates the next question, the .tomal file should have the links twice or thrice ( as per the site ), Or only once with the related variable(new ones) as per the combinations available in the site ?
Instead of 'sessionsURL' & 'appsURL', How about sessions&appsURL & similar?
